js/components/graph/Sidebar.js

```diff
@@ -1,204 +1,100 @@
 import React from "react";
 import PropTypes from "prop-types";
-import Focus from "./Focus";
 
 export default class Sidebar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       contentHidden: true,
```

Comment:
Okay, it looks like you have a good opportunity to simplify this component. Currently there are two states (`contentHidden` and `toggled`), which do essentially mean the same thing, and in fact it looks like `contentHidden` isn't really used.

Neither `contentHidden` or `toggled` are great names. Make a new state called `hidden` which is used to determine whether the sidebar is hidden or not, and update the rest of the code accordingly.